app/src/main/res/layout/fragment_details.xml

```diff
@@ -105,10 +105,18 @@
                 android:layout_below="@id/col2"
                 android:layout_marginStart="20dp"
                 android:layout_marginTop="40dp"
-                android:layout_marginEnd="20dp"
-                android:clickable="false"
-                android:filterTouchesWhenObscured="true" />
+                android:layout_marginEnd="20dp"/>
 
+        <View
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/col2"
```

Comment:
I guess you remove this line. Try it, works fine after removing this line then remove it.

Reply:
I can only remove layout_below attribute , if I remove height and width it is giving an error.

Reply:
Yes, I am talking about layout_below attribute only.